docs/node-operators/migration-guides/update-network-with-checkpoint.md

```diff
@@ -0,0 +1,265 @@
+* `<BACKUP-FOLDER>`: the folder where you store backups, e.g., `/home/vega/backups`
+* `<VISOR-BIN>`: the path to the Vega Visor binary, e.g., `/home/vega/bin/visor`
+* `<VEGA-BIN>`: the path to the Vega core binary, e.g., `/home/vega/bin/vega`
+* `<CHAIN-ID>`: new chain ID for network; it is required to pass as an argument for data-node, e.g., current chain ID on mainnet is: `vega-mainnet-0009`
+
+The following are placeholders for the PostgreSQL connection details for the data node - the ones you put in the data node `config.toml`.
+
+* `<VEGA-DB-USER>` - PostgreSQL user you create and put in the data node config
+* `<VEGA-DB-NAME>` - PostgreSQL database name
+
+We will refer to the above paths in the following guide. The sample paths given above are just examples. We recommend setting the paths that align with the conventions adopted by your organisation.
+
+
+## Study the changes between versions
+Before upgrading your node software, read the upgrading file in the Vega repo for a full list of the changes between the two versions, and review the breaking API changes.
+
+Review the **[release notes](../../releases/overview.md)** for a list of breaking API changes for each version from 0.54 onwards.
+
+Review the **[upgrading readme ↗](https://github.com/vegaprotocol/vega/blob/develop/UPGRADING.md)** with details on major updates.
+* [Repository changes ↗](https://github.com/vegaprotocol/vega/blob/develop/UPGRADING.md#repository-changes)
+* [Configuration changes ↗](https://github.com/vegaprotocol/vega/blob/develop/UPGRADING.md#configuration-changes)
+* [Command line changes ↗](https://github.com/vegaprotocol/vega/blob/develop/UPGRADING.md#command-line-changes)
+
+## Upgrade steps
+
+### 1. Stop the network
+At this point, validators need to choose and agree on the checkpoint that will be loaded into the network during the next restart, and stop the network as soon as everyone agrees on the selected checkpoint.
+
+The reason to quickly stop the network is to avoid producing more checkpoints and missing transactions after the restart (the transactions executed between when the selected checkpoint is produced, and the network is stopped).
+
+For testnet releases, the time requirement may be relaxed, but mainnet downtime must be limited as much as possible.
+
+1. Stop the running node
+    - Stop `vega` and optionally `data-node` if you are not running vegavisor: `sudo systemctl stop vega && sudo systemctl stop data-node`
+    - Stop `vegavisor` if you are running your node under the `vegavisor`: `sudo systemctl stop vegavisor`
+2. Verify the node has been stopped:
+    - Vega: `systemctl status vega`
+    - Optionally data-node: `systemctl status data-node`
+    - Optionally vegavisor: `systemctl status vegavisor`
+
+
+### 2. Create backup
+
+```bash
+mkdir -p <BACKUP-FOLDER>/v0.53.0/wallets;
+mkdir -p <BACKUP-FOLDER>/v0.53.0/core-state;
+mkdir -p <BACKUP-FOLDER>/v0.53.0/tm-state;

```

Comment:
maybe update all directories to `v0.68.0` (here and in the following lines)

Reply:
That's valid point. Thank you